backports.cpp

```diff
@@ -26,3 +27,15 @@ void LLVMGlobalObjectAddMetadata(LLVMValueRef Global, unsigned KindID, LLVMMetad
   llvm::GlobalObject *O = llvm::unwrap<llvm::GlobalObject>(Global);
   O->addMetadata(KindID, *N);
 }
+
+LLVMMetadataRef
+LLVMDIBuilderCreateTypedef_LLVM10(LLVMDIBuilderRef Builder, LLVMMetadataRef Type,
```

Comment:
I personally don't really like the LLVM version in the symbol name (it will be outdated in half a year). Incompatibilities have usually been resolved with a new `LLVMGoXXX` function so would prefer to have that in this case too.

Reply:
Changed the names.